modules/css/button/stories.tsx

```diff
@@ -30,7 +30,7 @@ const spaceButton = css({
   },
 });
 
-class IconDemo extends Component {
+class IconDemo extends Component<React.PropsWithChildren<any>> {
```

Comment:
My best guess with this change is required because of https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210

`children` is no longer implicit with many React components. I don't think the `any` is what we want though - it means ANY props are valid.